### ISSUE TITLE
Tag for deployment

### ISSUE THREAD

**Issue by chimmelb** [jjongsma/grunt-eb-deploy#2]
If there is a way to specify arguments for a grunt task from the command line, it'd be nice to specify a unique tag in addition to the date. This way I could put my current version information in there, and easily identify it in the tag listing.

I'd like to run `grunt ebDeploy:env:tag=v0.0.234` and see a build created like `my-app-name-20150218--558486--v0.0.234`

**Comment by jjongsma** [comment]
The app version name already contains the Git commit tag after the date, if that helps. I'm not aware of anyway to pass arbitrary values into grunt plugins (though I'm not a Grunt plugin expert).

**Comment by jjongsma** [comment]
Looks like it might be possible with this:

http://stackoverflow.com/questions/20127586/grunt-command-line-parameters

**Comment by chimmelb** [comment]
Yeah, that'd be cool. My application name is present, and it'd be nice to add a version or tag for each push.
